### PR TITLE
Add some new misspellings

### PR DESCRIPTION
**Pull request by Jeremiah-England**
Ref: https://github.com/codespell-project/codespell/issues/2868

Sorry it took so long! Let me know if you see anything I need to update.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -43261,6 +43303,8 @@ traget->target
 trageted->targeted
 trageting->targeting
 tragets->targets
+traid->triad, trade,
+traids->triads, trades,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:43307]:
```
traid->triad, trade, raid,
traids->triads, trades, raids,
```

**Review by DimitriPapadopoulos — COMMENTED** [review]
A few suggestions.

**Comment by Jeremiah-England** [comment]
Thanks @DimitriPapadopoulos.

I am not sure what your normal workflow for resolving threads is. I incorporated all your suggestions but I left the threads for you to resolve in case you want to double check and resolve as you go.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
Thank you for your contribution @Jeremiah-England.

**Pull request merged**